ui/hooks/useCarouselManagement/useCarouselManagement.test.ts

```diff
@@ -449,4 +487,25 @@ describe('useCarouselManagement', () => {
       expect(mockUpdateSlides).toHaveBeenCalled();
     });
   });
+
+  describe('Smart account upgrade slide', () => {
+    it('should not be displayed if solana address is selected', () => {
+      jest.spyOn(AccountUtils, 'isSolanaAddress').mockReturnValue(true);
+      renderHook(() => useCarouselManagement({ testDate: validTestDate }));
+
+      const updatedSlides = mockUpdateSlides.mock.calls[0][0];
+
+      expect(updatedSlides).toStrictEqual([
+        { ...SWEEPSTAKES_SLIDE, dismissed: false },
+        { ...FUND_SLIDE, undismissable: true },
+        ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)
+        BRIDGE_SLIDE,
+        ///: END:ONLY_INCLUDE_IF
+        CARD_SLIDE,
+        CASH_SLIDE,
+        MULTI_SRP_SLIDE,
+        SOLANA_SLIDE,
+      ]);
```

Comment:
cc @jpuri `BACKUPANDSYNC_SLIDE` is removed from this assertion due to introduced after release cut from: 
https://github.com/MetaMask/metamask-extension/pull/32129